### PR TITLE
Workflow Automated Test - LRQA-70679 | master 

### PR DESCRIPTION
**Pull request by alessandromalves**
https://issues.liferay.com/browse/LRQA-70679

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase [hunk 1]
```diff
@@ -59,12 +59,44 @@ definition {
...
+			kfProcessName = "Kaleo Forms Process",
+			workflowDefinitionTitle = "Single Approver");
+		
+		KaleoFormsAdmin.chooseWorkflow(
+			kfProcessName = "Kaleo Forms Process",
+			workflowDefinitionTitle = "Workflow Test");
+
+		AssertTextNotEquals(
+			locator1 = "KaleoFormsAdminEditProcess#SELECTED_WORKFLOW",
+			value1 = "Single Approver");
```
Comment (lucasperj) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase:94]: You can use the macro created at LRQA-70678 | master #81

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase [hunk 2]
```diff
@@ -93,12 +93,39 @@ definition {
...
+			kfProcessName = "Kaleo Forms Process",
+			workflowDefinitionTitle = "Single Approver");
+
+		KaleoFormsAdmin.chooseWorkflow(
+			kfProcessName = "Kaleo Forms Process",
+			workflowDefinitionTitle = "Workflow Test");
+
+		KaleoFormsAdmin.viewNotSelectedWorkflow(workflowDefinitionTitle = "Single Approver");
+
+		KaleoFormsAdmin.next();
```
Comment (alessandromalves) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/tests/KaleoFormsNewProcess.testcase:128]: remove

**Review by lucasperj — APPROVED** [review]

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by alessandromalves** [comment]
ci:test:sf

**Pull request closed without merging**